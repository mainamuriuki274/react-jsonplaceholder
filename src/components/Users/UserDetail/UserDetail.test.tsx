import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { UserDetail } from './index';

describe('User Detail Component', () => {
  it('it renders', () => {
    render(<UserDetail label="Test Label" value="Test Value" />);
    expect(screen.getByText('Test Label')).toBeInTheDocument();
    expect(screen.getByText('Test Value')).toBeInTheDocument();
  });

  it('it matches snapshot', () => {
    const tree = renderer
      .create(<UserDetail label="Test Label" value="Test Value" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

export {};
