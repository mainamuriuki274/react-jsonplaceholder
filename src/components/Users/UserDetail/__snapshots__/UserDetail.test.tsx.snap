// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Detail Component it matches snapshot 1`] = `
<p>
  <span
    className="label"
  >
    Test Label
  </span>
  <span
    className="value"
  >
    Test Value
  </span>
</p>
`;
